.github/wokflows/cypress-parallel.yml

```
@@ -0,0 +1,45 @@
+name: Cypress tests
+
+on: [push, pull_request]
```

```suggestion
on: [pull_request]
```

It looks like we can run this just on `pull_request`.

https://github.com/cypress-io/github-action/issues/134

Which supports what we found on Chris's repo.

---

updated